src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
+    alpr = Alpr(options.country, options.config, options.runtime_data)
```

Comment:
I don't mind if you want to create a separate file to test the CV bindings, however it seemed to me to make perfect sense to update the existing file, because you are simply augmenting the existing implementation to test a slightly different case.